### PR TITLE
New nb_NO strings

### PR DESCRIPTION
**Pull request by comradekingu**
For https://github.com/falzonv/discreet-launcher/issues/80

### CODE REVIEW

File: app/src/main/res/values-nb-rNO/strings_settings.xml [hunk 1]
```diff
@@ -70,12 +71,12 @@
...
         <item>Ingen (skjult)</item>
-        <item>24-timersklokke</item>
-        <item>12-timersklokke</item>
-        <item>12-timersklokke + AM/PM</item>
+        <item>24-timers</item>
+        <item>12-timers</item>
+        <item>12-timers + AM/PM</item>
         <item>Analog</item>
         <item>Dato og klokkeslett</item>
         <item>Full dato og klokkeslett</item>
```
Comment (falzonv) [inline app/src/main/res/values-nb-rNO/strings_settings.xml:82]: The 2 lines below this one should also be translated:
```
<item>Date without time</item>
<item>Full date without time</item>
```

**Comment by falzonv** [comment]
Aside from the 2 missing strings (below), everything looks good so I will merge this PR, thank you for the update!

When you get some time, could you please write the missing strings here as a comment?

```
      <!-- Clock format -->
      <string name="set_clock_format">Format</string>
      <string-array name="set_clock_format_entries">
          <item>None (hide)</item>
          <item>24 hours</item>
          <item>12 hours</item>
          <item>12 hours + AM/PM</item>
          <item>Analog</item>
          <item>Date and time</item>
          <item>Full date and time</item>
NEW       <item>Date without time</item>
NEW       <item>Full date without time</item>
      </string-array>
```

**Comment by comradekingu** [comment]
@falzonv 
```
<item>Dato uten klokkeslett</item>
<item>Full dato uten klokkeslett</item>
```

**Comment by falzonv** [comment]
Thank you, integrated!

**Pull request merged**